azure-pipelines.yml

```diff
@@ -97,7 +97,7 @@ stages:
               graphviz \
               inkscape \
               libcairo2 \
-              libgirepository-1.0.1 \
+              libgirepository-1.0-1 \
```

Comment:
No idea what this is used for or why the packaged made a minor name change like this, but the old name was broken on 20.04.